src/main/java/org/opendevstack/provision/authentication/SimpleCachingGroupMembershipManager.java

```diff
@@ -77,7 +77,7 @@ public List getMemberships(String user) throws RemoteException, InvalidAuthoriza
 				basicCache.addGroupToUser(user, group);
 				logger.debug("add to user: " +  user + " > " + group);
 			}
-			logger.info("add to cache (" +user+ ") took: " + 
+			logger.debug("add to cache (" +user+ ") took: " + 
 				(System.currentTimeMillis() - startTime) + "ms");

```

Comment:
avoid String concatenation in log statements as they will always be evaluated.
rather do:
`logger.debug("add to cache ({}) took: {} ms", user, (System.currentTimeMillis() - startTime));`